src/backend/web/routes/health.js

```diff
@@ -4,7 +4,7 @@ const healthcheck = require('express-healthcheck');
 // We want the current version from package.json
 const { version } = require('../../../../package.json');
 // As well as the current git sha
-const sha = process.env.GIT_COMMIT || 'undefined';
+const sha = 'master';
```

Comment:
```suggestion
const sha = process.env.GIT_COMMIT || 'master';
```

Reply:
This env value doesn't exist anymore in any of our env files, should we really be referring to this still?

Reply:
We add it to the `env` file used for deployment at build time [here](https://github.com/Seneca-CDOT/telescope/blob/da0b0f664743f54a22e0f2605aba82aed11d0425/tools/autodeployment/deploy.sh#L34)

Reply:
Ahh, figured it gets used in deployment. Will add it back